portable/GCC/ARM_CM23_NTZ/non_secure/portmacrocommon.h

```diff
@@ -144,13 +144,13 @@ extern void vClearInterruptMask( uint32_t ulMask ) /* __attribute__(( naked )) P
 #endif
 
 /* MPU regions. */
-#define portPRIVILEGED_FLASH_REGION                   ( 0UL )
-#define portUNPRIVILEGED_FLASH_REGION                 ( 1UL )
+#define portPRIVILEGED_FLASH_REGION                   ( 6UL )
+#define portUNPRIVILEGED_FLASH_REGION                 ( 5UL )
 #define portUNPRIVILEGED_SYSCALLS_REGION              ( 2UL )
```

Comment:
Should this lie smack dab in the middle of the configurable regions? Why not have it at either the beginning or end given it has to be configured if the MPU is enabled?

Reply:
This was done as the CM0+ allows overlapping regions with the highest numbered region taking precedence. The V8M does not allow overlapping regions.